app/src/main/java/com/ankihighlights/android/model/Highlight.kt

```diff
@@ -0,0 +1,33 @@
+package com.ankihighlights.android.model
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class HighlightData(
+    val word: String,
+    val context: String,
+    val timestamp: Long
+)
+
+@Serializable
+data class HighlightResponse(
```

Comment:
Made the changes. But I am going not yet resolve this cause I want to get another opinion about this after I make the commit broken out files. Specifically the HighlightResponse and HighlightDataWrapper. I dont like these Serializables because its not exactly how it worked in my Spring Boot backend I wrote last year.